src/components/TextLink.js

```diff
@@ -32,19 +32,29 @@ const defaultProps = {
 
 const TextLink = (props) => {
     const additionalStyles = _.isArray(props.style) ? props.style : [props.style];
+
+    const handlePress = (e) => {
```

Comment:
Please change the name of this. Functions should be named for what they do, not what they handle.

Reply:
Also, missing jsdocs for this callback

Reply:
done